### PR TITLE
Create 2017-11-14-Tiny-servant.markdown

### PR DESCRIPTION
**Pull request by Eikrem63**

### CODE REVIEW

File: _posts/2017-11-14-Tiny-servant.markdown [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "Tiny Servant"
+date:   2017-11-14
+source: Xanathar.169
```
Comment (Traneptora) [inline _posts/2017-11-14-Tiny-servant.markdown]: It's on Page 168.

File: _posts/2017-11-14-Tiny-servant.markdown [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+**Range**: Touch
+
+**Components**: V, S
+
+**Duration**: 8 hours
+
+You touch one Tiny, nonmagical object that isn't attached to another object or a surface and isn’t being carried by another creature. The target animates and sprouts little arms and legs, becoming a creature under your control until the spell ends or the creature drops to 0 hit points. See the stat block for its statistics.
+
+As a bonus action, you can mentally command the creature if it is within 120 feet of you. (If you control multiple creatures with this spell, you can command any or all of them at the same time, issuing the same command to each one.) You decide what action the creature will take and where it will move during its next turn or, you can issue a simple, general command, such as to fetch a key, stand watch, or stack some books, If you issue no commands, the servant does nothing other than defend itself against hostile creatures. Once given an order, the servant continues to follow that order until its task is complete.
```
Comment (Traneptora) [inline _posts/2017-11-14-Tiny-servant.markdown:21]: Misplaced comma: "during its next turn, or you can" not "during its next turn or, you can"

Comment (Traneptora) [inline _posts/2017-11-14-Tiny-servant.markdown:21]: Incorrect punctuation: "or stack some books. If you issue" not "or stack some books, If you issue"

File: _posts/2017-11-14-Tiny-servant.markdown [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+**Casting Time**: 1 minute
+
+**Range**: Touch
+
+**Components**: V, S
+
+**Duration**: 8 hours
+
+You touch one Tiny, nonmagical object that isn't attached to another object or a surface and isn’t being carried by another creature. The target animates and sprouts little arms and legs, becoming a creature under your control until the spell ends or the creature drops to 0 hit points. See the stat block for its statistics.
```
Comment (Traneptora) [inline _posts/2017-11-14-Tiny-servant.markdown:19]: Add the stat block as a Markdown table.

Comment (Eikrem63) [reply _posts/2017-11-14-Tiny-servant.markdown:19]: Where do you want me to put it? just at the end after the "higher levels" line?

Comment (Traneptora) [reply _posts/2017-11-14-Tiny-servant.markdown:19]: The best place would be right below this paragraph.

**Comment by Traneptora** [comment]
See #60

**Pull request closed without merging**